README.md

```diff
@@ -1,15 +1,47 @@
-#Repository for Innovation Labs infographic website
-To run using npm http-server
+# Infographic Infrastructure Automation Demo
+
+## Prerequisistes
+* NodeJS
+* npm
+
+## Running
 Run: 
-```js
-npm install -g http-server
-http-server website
+```bash
+npm install
+npm start
 ```
 
-This will start up a local http server using the `website` directory as the root context of the server.
+This will start up a local http server to host the fron using the `website` directory as the root context of the front end application. This local devleopment server will also host the API. In OpenShift-based environments, the front end and back end are hosted as separate applications, using the templates in the `templates` directory.
```

Comment:
I know the templates are mentioned here, but would like maybe give them their own dedicated section. That's the primary way to run this.